backend/src/main/java/com/jobwait/domain/Answers.java

```diff
@@ -9,13 +11,65 @@
+        public ValidEducationLevel educationLevel;
+
+        public Answers() {
+                this.jobAcceptDate = null;
+                this.jobSearchStartDate = null;
+                this.workModel = null;
+                this.workContract = null;
+                this.jobApplicationCount = null;
+                this.jobTitle = null;
+                this.yearsOfProExperience = null;
+                this.educationLevel = null;
+        }
+
+        public Answers(OffsetDateTime jobAcceptDate,
+                        OffsetDateTime jobSearchStartDate,
+                        ValidWorkModel workModel,
+                        ValidWorkContract workContract,
+                        Integer jobApplicationCount,
+                        String jobTitle,
+                        Integer yearsOfProExperience,
+                        ValidEducationLevel educationLevel) {
+                this.jobAcceptDate = jobAcceptDate;
+                this.jobSearchStartDate = jobSearchStartDate;
+                this.workModel = workModel;
+                this.workContract = workContract;
+                this.jobApplicationCount = jobApplicationCount;
+                this.jobTitle = jobTitle;
+                this.yearsOfProExperience = yearsOfProExperience;
+                this.educationLevel = educationLevel;
+        }
+
+        public Answers(List<Answer> listOfAnswers) {
+                this.jobAcceptDate = (OffsetDateTime) listOfAnswers.get(0).getValue();
+                this.jobSearchStartDate = (OffsetDateTime) listOfAnswers.get(1).getValue();
+                this.workModel = (ValidWorkModel) listOfAnswers.get(2).getValue();
+                this.workContract = (ValidWorkContract) listOfAnswers.get(3).getValue();
+                this.jobApplicationCount = (Integer) listOfAnswers.get(4).getValue();
+                this.jobTitle = (String) listOfAnswers.get(5).getValue();
+                this.yearsOfProExperience = (Integer) listOfAnswers.get(6).getValue();
+                this.educationLevel = (ValidEducationLevel) listOfAnswers.get(7).getValue();
+        }
+
+        public List<Answer> listOfAnswers() {
+                return Arrays.asList(
+                                new Answer<OffsetDateTime>(this.jobAcceptDate, "jobAcceptDate"),
+                                new Answer<OffsetDateTime>(this.jobSearchStartDate, "jobSearchStartDate"),
+                                new Answer<ValidWorkModel>(this.workModel, "workModel"),
+                                new Answer<ValidWorkContract>(this.workContract, "workContract"),
+                                new Answer<Integer>(this.jobApplicationCount, "jobApplicationCount"),
+                                new Answer<String>(this.jobTitle, "jobTitle"),
+                                new Answer<Integer>(this.yearsOfProExperience, "yearsOfProExperience"),
+                                new Answer<ValidEducationLevel>(this.educationLevel, "educationLevel"));
+        }
```

Comment:
i hate this so much....

Reply:
can we put some of these things in a list? or set some default values or something? i dont want to change 20 lines of code whenever i want to add/remove a question